docs/gettingstarted/installation.rst

```diff
@@ -151,12 +151,18 @@ Using Amazon Web Services
 .. note:: If you are not familiar with EC2 you may want to `read more about the terminology and concepts <https://fedoraproject.org/wiki/User:Gholms/EC2_Primer>`_ used in this document.
           You can also refer to `the full documentation for interacting with EC2 from Amazon Web Services <http://docs.amazonwebservices.com/AWSEC2/latest/GettingStartedGuide/>`_.
 
-#. Choose an AMI for your EC2 Instance
-
-   * Visit https://fedoraproject.org/en/get-fedora#clouds
-   * In the EC2 **Fedora 20** section choose your local region and **64 Bit** and click the "Launch it!" button.
-
-#. Configure the AMI
+#. Choose a region for your EC2 instance
```

Comment:
I tried to clarify the wording.  Unfortunately Sphinx makes it pretty hard to put `target=...` into links (seems like you can only do it by embedding raw html in the document).  Maybe with the new wording it's clear enough that folks can open the right link in a new tab/window now.